src/components/user/ProposalsPeopleTable.tsx

```diff
@@ -390,6 +397,32 @@ const ProposalsPeopleTable: React.FC<PeopleTableProps> = (props) => {
               </Alert>
             )}
           </div>
+
+          {!isLargeScreen && (
+            <div className={classes.mobileUpdateButton}>
+              {props.selection && (
+                <ActionButtonContainer>
+                  <div className={classes.verticalCentered}>
+                    {selectedParticipants.length} user(s) selected
+                  </div>
+                  <Button
+                    type="button"
+                    onClick={() => {
+                      if (props.onUpdate) {
```

Comment:
Here too!
The onClick handler could be moved outside the HTML part into some `onClickHandler`, and then referenced in both large and small screens.